test/jdk/java/awt/datatransfer/HTMLDataFlavors/ManualHTMLDataFlavorTest.java

```diff
@@ -22,24 +22,32 @@
  */
 
 /*
-  test
+  @test
   @bug 7075105
   @summary WIN: Provide a way to format HTML on drop
```

Comment:
Do we add `@requires (os.family == "windows")` since the test summary indicates Windows issue?

Reply:
It is valid on all platforms. In fact if you read the instructions you'll see it says
"     On Mac OS X do NOT use Safari, it does not provide the needed DataFlavor,"